tests/tests.rs

```diff
@@ -1911,3 +1911,27 @@ fn serde() {
         "invalid value: integer `-1`, expected u128",
     );
 }
+
+#[cfg(all(feature = "borsh", feature = "std"))]
+#[test]
+fn borsh() {
+    use borsh::schema::BorshSchemaContainer;
+    use borsh::{BorshDeserialize, BorshSerialize};
+    let mut buf = borsh::__private::maybestd::vec::Vec::new();
```

Comment:
Never seen borsh being used before, but it seems odd to require a __private namespace in public code. I'm assuming this could break anytime if they decide to move things around?

Reply:
ok, will replace with Vec. seems just used suggestion of RA. that just core vs std Vec reference.